public/zt.js

```diff
@@ -30,44 +32,87 @@ function generateDataElements(data, listElement,formElement) {
       element.innerHTML += `<span class="ms-auto" id="${member.config.id}-ipAssigment">${member.config.ipAssignments[0]}</span>`;
 
       listElement.append(element)
+
     }
-    else{
 
-      
+    //Generate options for form
+    if (!member.hidden) {
+      let selectElement = $('#Select-form');
+
+
+      //Checking if member has name if not assigning member id as his shown name
+      let displayName = member.name + " " + member.description;
+      if (displayName === " ") {
+        displayName = member.config.id;
+      }
+
+      //Coloring diffrent options for easier seeing (green - authorised, red - unauthorised)
+      if (member.config.authorized) {
+        selectElement.innerHTML += `<option style="color:Green" value="${[member.config.id, member.name, member.description]}">${displayName}</option>`
+      }
+      else {
+        selectElement.innerHTML += `<option style="color:Red" value="${[member.config.id, member.name, member.description]}">${displayName}</option>`
```

Comment:
styling should be handled by [bootstrap](https://getbootstrap.com/docs/5.3/utilities/colors/#colors) to keep consistent styling.